core/src/main/resources/error/error-classes.json

```diff
@@ -602,6 +602,24 @@
     ],
     "sqlState" : "42000"
   },
+  "NON_LAST_MATCHED_CLAUSE_OMIT_CONDITION" : {
+    "message" : [
+      "When there are more than one MATCHED clauses in a MERGE statement, only the last MATCHED clause can omit the condition."
+    ],
+    "sqlState" : "42000"
+  },
+  "NON_LAST_NOT_MATCHED_BY_SOURCE_CLAUSE_OMIT_CONDITION" : {
+    "message" : [
+      "When there are more than one NOT MATCHED BY SOURCE clauses in a MERGE statement, only the last NOT MATCHED BY SOURCE clause can omit the condition."
+    ],
+    "sqlState" : "42000"
+  },
+  "NON_LAST_NOT_MATCHED_CLAUSE_OMIT_CONDITION" : {
```

Comment:
is it more accurate to name it `NON_LAST_NOT_MATCHED_BY_TARGET_CLAUSE_OMIT_CONDITION `